fe/src/main/java/org/apache/doris/qe/GlobalVariable.java

```diff
@@ -56,6 +56,10 @@
     @VariableMgr.VarAttr(name = "query_cache_size")
     private static long queryCacheSize = 1048576;
 
+    // the flags passed to Backends
+    @VariableMgr.VarAttr(name = "is_set_default_rowset_to_beta")
+    public static boolean isSetDefaultRowsetToBeta = false;
```

Comment:
If this flag is set here, when there are multiple frontends, user should set all of them?

Reply:
Global variable can be synced to other frontends automatically now.

Reply:
I change it to sessionvariable type.